task/upload.go

```diff
@@ -244,6 +245,19 @@ func processCatalystCallback(tctx *TaskContext, callback *clients.CatalystCallba
 			})
 		case OutputNameIPFSSourceMP4:
 			assetSpec.Storage.IPFS.CID = output.Manifest
+		case OutputNameAssetMP4:
+			if len(output.Videos) != 1 {
+				return nil, fmt.Errorf("unexpected number of videos in source MP4 output: %d", len(output.Videos))
+			}
+			video := output.Videos[0]
```

Comment:
Need to loop over all videos.

Reply:
Right! I haven't pushed, it's there now. Thanks for the catch